state/azure/cosmosdb/cosmosdb.go

```diff
@@ -249,7 +249,18 @@ func (c *StateStore) Get(req *state.GetRequest) (*state.GetResponse, error) {
 	}
 
 	if items[0].IsBinary {
-		bytes, _ := base64.StdEncoding.DecodeString(items[0].Value.(string))
+		if items[0].Value == nil || len(items[0].Value.(string)) == 0 {
```

Comment:
You may need to do a check if the type assertion is possible, before asserting as `(string)` right away

Reply:
It is always string because even binary data is saved as a base64 encoded string. This is why the decoding below always applies - except when the value in the database is literally null.